Crunchyroll-Downloader/App.config

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.1"/>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>

```

Comment:
Please, don't downgrade the .NET Framework version.

Reply:
Oops. Sorry. I didn't see that.